src/content/en/2020/http2.md

```diff
@@ -126,52 +185,54 @@ Figure 22.6 shows the fraction of HTTP/2 or gQUIC requests per page, ordered by
   sheets_gid="419557288",
   sql_file="http2_%_1st_party_vs_3rd_party_by_type.sql"
 )
 }}
 
-Figure 22.9 looks at protocol usage by the third party category. Ads, analytics, Content Delivery Network resources (CDN), and tag-managers are predominantly served over HTTP/2 or gQUIC. Customer-success and marketing content is more likely to be served over HTTP/1.
+{# TODO(authors, analysts): Is this breakdown specifically looking at mobile? I don't see a notice that all data is mobile unless otherwise noted, so it's worth clarifying here. #}
+Figure 22.9 looks at protocol usage by the third party category. Ads, analytics, content delivery network (CDN) resources, and tag-managers are predominantly served over HTTP/2 or gQUIC. Customer-success and marketing content is more likely to be served over HTTP/1.
 
 {{ figure_markup(
   image="http2-third-party-http2-usage-by-category.png",
-  caption="The fraction of known third-party HTTP/2 or gQUIC requests by category per web site.",
-  description="A bar chart comparing the fraction of third-party HTTP/2 or gQUIC requests by category. All third-party requests for all web sites are served over HTTP/2 or gQUIC for 95% of tag managers, 90% of analytics and CDN, 80% of ads, social, hosting and utility, 40% of marketing and 30% of customer-success.",
+  caption="The fraction of known third-party HTTP/2 or gQUIC requests by category per website.",
+  description="A bar chart comparing the fraction of third-party HTTP/2 or gQUIC requests by category. All third-party requests for all websites are served over HTTP/2 or gQUIC for 95% of tag managers, 90% of analytics and CDN, 80% of ads, social, hosting and utility, 40% of marketing and 30% of customer-success.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1419102835&format=interactive",
   sheets_gid="1059610651",
   sql_file="http2_3rd_party_by_types.sql"
 )
 }}
 
-### HTTP/2 Server support
+### Server support
 
-Browser auto-update mechanisms are a driving factor for client-side adoption of new Web standards. It’s [estimated](https://caniuse.com/http2) that over 97% of global users support HTTP/2, up slightly from 95% measured last year.
+Browser auto-update mechanisms are a driving factor for client-side adoption of new web standards. It's [estimated](https://caniuse.com/http2) that over 97% of global users support HTTP/2, up slightly from 95% measured last year.
 
 Unfortunately, the upgrade path for servers is more difficult, especially with the requirement to support TLS. For mobile and desktop, Figure 22.10 shows the breakdown of HTTP server headers returned for sites that support HTTP/2 and HTTP/1.
 
+{# TODO(analysts, authors): Why is this figure in terms of the number of websites, rather than percent? Should the server values also be capitalized accordingly? (eg Cloudflare rather than cloudflare) #}
```

Comment:
@dotjs @bazzadp

Reply:
This was @dotjs but I actually think I agree with him here.

I think a percentage in terms of all sites, or even relative to that particular server, would be misleading and lose the fact that nginx and and apache have way more sites than gse for example.

As to the capitalisation these are the server headers (as stated in the line above the graph) rather than company names and looks like this year we normalised to avoid any case issues. I think it's fine as is.

Reply:
This was discussed with the authors/analysts. Using the absolute number of websites allows the reader to see the proportion of h2 vs h1 for each server and also see the relative proportion of websites served per sever.